roles/network_validated_content/templates/CHANGELOG.rst.j2

```diff
@@ -0,0 +1,5 @@
+====================================
```

Comment:
We should not generate this. It should be coming from antsichaut or antsibull-changelog.